test/ctia/auth/jwt_test.clj

```diff
@@ -33,45 +40,45 @@
 
 
 (deftest scopes-to-capapbilities-test
-  (is (= "private-intel" sut/entity-root-scope)
+  (is (= "private-intel" @sut/entity-root-scope)
```

Comment:
setting default properties configuration should be done on `ctia-default.properties` and not hardcoded, doing this will make this test useless.